bitnami/influxdb/README.md

```diff
@@ -169,6 +169,19 @@ The following tables lists the configurable parameters of the InfluxDB chart and
+| `backup.retentionDays`                      | retention time in days for backups                                                                                                                                                        | `10`                                                    |
+| `backup.annotations`                        | backup annotations                                                                                                                                                                        | `{}`                                                    |  
+| `backup.upload.google.enabled`              | enable upload to google storage bucket                                                                                                                                                    | `false`                                                 |
+| `backup.upload.google.serviceAccountSecret` | service account secret name                                                                                                                                                               | `"influxdb-backup-gcs-key"`                             |
+| `backup.upload.google.serviceAccountSecretKey` | service account secret key                                                                                                                                                             | `"key.json"`                                            |
+| `backup.upload.google.bucketname`           | google storage bucket name                                                                                                                                                                | `"gs://bucket/influxdb"`                                |
+| `backup.upload.azure.enabled`               | enbale upload to azure storage                                                                                                                                                            | `false`                                                 |
+| `backup.upload.azure.storageAccountSecret`  | storage account secret                                                                                                                                                                    | `"influxdb-backup-azure-key"`                           |
+| `backup.upload.azure.destination_container` | destination container                                                                                                                                                                     | `"influxdb-container"`                                  |
+| `backup.upload.azure.destination_path`      | destination path                                                                                                                                                                          | `""`                                                    |
```

Comment:
If you check the parameter `existingSecret` and `adminUser.pwd`, `writeUser.pwd`, etc., you'll find out that we normally provide two ways to pass "credentials"
- Directly setting in the values.yaml (through a parameter)
- Taken them from existing secrets

It'd be nice if we follow the same structure here. 

My proposal:

|  Parameter    |   Description                         |   Default  |
| --- | --- | --- |
| `backup.enabled` | Enable periodically creating InfluxDB backup(s) | `false`  |
| `backup.cronjob.schedule` | Cronjob schedule | `"0 2 * * *"` |
| `backup.cronjob.historyLimit` | Number of successful finished jobs to retain | `1` |
| `backup.cronjob.podAnnotations` | Pod annotations for cronjob pods | `{}` |
| `backup.providers` | Array with Cloud Storage Providers (options: aws, azure, google, ...)  | `[]` |
| `backup.azure.storageAccount` | When using the Azure provider, specify content for `storageAccount.json` | `nil` |
| `backup.azure.storageAccountSecret` | When using the Azure provider, name of existing secret containing the `storageAccount.json` file | `nil` |
| `backup.azure.storageAccountSecretKey` | When using the Azure provider with an existing secret, specify the key name | `storageAccount.json` |
| `backup.azure.destination` | When using the Azure provider, destination container where to upload backups | `influxdb-container`|
| `backup.azure.destinationPath` | When using the Azure provider, path within the destination container where to upload backups | `nil`|
| `backup.google.serviceAccountKey` | When using the Google provider, specify the service account key JSON file | `nil` |
| `backup.google.serviceAccountSecret` | When using the Google provider, name of existing secret containing the the service account key | `nil` |
| `backup.google.serviceAccountSecretKey` | When using the Google provider with an existing secret, specify the key name | `key.json` |
| `backup.google.bucketname` | When using the Google provider, specify the storage bucket name                                                                                                                                 | `"gs://bucket/influxdb"` |

Reply:
Did you see my suggestion here?